.github/workflows/windows_deps.yml

```diff
@@ -0,0 +1,649 @@
+
+      - name: 'Package dependencies'
+        if: success()
+        working-directory: ${{ github.workspace }}/windows/obs-dependencies-${{ matrix.arch }}
+        run: |
+          CURRENT_DATE=$(date +"%Y-%m-%d")
+          FILE_NAME="windows-cross-deps-${CURRENT_DATE}-${{ matrix.arch }}.tar.xz"
+          echo "FILE_NAME=${FILE_NAME}" >> $GITHUB_ENV
+
+          cp -R ${{ github.workspace }}/licenses .
+
+          XZ_OPT=-T0 tar -cJf "${FILE_NAME}" *
+
+          mv ${FILE_NAME} ..
+
+      - name: 'Publish Build Artifacts'
+        if: ${{ success() && (github.event_name != 'pull_request' || env.SEEKING_TESTERS == '1') }}
+        uses: actions/upload-artifact@v2.2.4
+        with:
+          name: ${{ env.FILE_NAME }}
+          path: ${{ github.workspace }}/windows/${{ env.FILE_NAME }}
+
+  windows-deps-build-native:
+    name: '01 - Build Windows dependencies (native)'
+    runs-on: [windows-2019]
+    strategy:
+      matrix:
+        arch: ['32', '64']
+    env:
+      CACHE_REVISION: '01'
+      MBEDTLS_VERSION: '2.24.0'
+      MBEDTLS_HASH: '523f0554b6cdc7ace5d360885c3f5bbcc73ec0e8'
+      CMOCKA_VERSION: '1.1.5-git'
+      CMOCKA_HASH: '9c114ac31a33217cf003bbb674c1aff7bb048917'
+      FREETYPE_VERSION: '2.10.4'
+      FREETYPE_HASH: '6a2b3e4007e794bfc6c91030d0ed987f925164a8'
+      CURL_VERSION: '7.73.0'
+      CURL_HASH: '315ee3fe75dade912b48a21ceec9ccda0230d937'
+      RNNOISE_VERSION: '90ec41ef659fd82cfec2103e9bb7fc235e9ea66c'
+      RNNOISE_HASH: '90ec41ef659fd82cfec2103e9bb7fc235e9ea66c'
+      SPEEXDSP_VERSION: '1.2rc3-git'
+      SPEEXDSP_HASH: '20ed3452074664ad07e380e51321b148acebdf20'
+      DETOURS_VERSION: 'v4.0.1'
+      DETOURS_HASH: 'e4bfd6b03e50de46b47abfbd1e46b384f0c5f833'
+      LUAJIT_VERSION: 'v2.0.5'
+      LUAJIT_HASH: '0bf80b07b0672ce874feedcc777afe1b791ccb5a'
+      PYTHON_VERSION: '3.6.2'
+      PYTHON_HASH: ''
+      VULKAN_VERSION: '1.2.131.2'
+      VULKAN_HASH: '49d515f091d69c005a9e52e69829d28383fe157a764caebaa3dbc4b8d9bb383d'
+      NASM_VERSION: '2.15.01'
+      NASM_HASH: ''
+      SWIG_VERSION: '3.0.12'
+      SWIG_HASH: '21ce6cbe297a56b697ef6e7e92a83e75ca41dedc87e48282ab444591986c35f5'
+    defaults:
+      run:
+        shell: pwsh
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v2.3.4
+
+      - name: 'Check for Github Labels'
+        if: github.event_name == 'pull_request'
+        shell: bash
+        run: |
+          if test -n "$(curl -H "Authorization: Bearer ${{ secrets.GITHUB_TOKEN }}" -s "${{ github.event.pull_request.url }}" | jq -e '.labels[] | select(.name == "Seeking Testers")')"; then
+            echo "SEEKING_TESTERS=1" >> $GITHUB_ENV
+          else
+            echo "SEEKING_TESTERS=0" >> $GITHUB_ENV
+          fi
+
+          echo "CACHE_DATE=$(date +"%Y-%m-%d")" >> $GITHUB_ENV
+
+      - name: 'Setup build environment'
+        shell: bash
+        run: |
+          mkdir -p windows_native_build_temp/win${{ matrix.arch }}/bin
```

Comment:
Same reasoning as above.  If these were in build scripts to be run on Windows, I would have used PowerShell, but since it's on GHA and Bash is available, I went with that, in part because fighting with Test-Path and New-Item can be a bit of a pain.